MaxMind.MinFraud/Request/CustomInputs.cs

```diff
@@ -23,6 +23,12 @@ private CustomInputs()
         {
         }
 
+        /// <summary>
+        ///     The custom inputs to be sent to the web service.
+        /// </summary>
+        [JsonExtensionData]
+        public IDictionary<string, object> Inputs => new ReadOnlyDictionary<string, object>(_inputs);
```

Comment:
Unfortunately, `System.Text.Json` doesn't support making this an `IReadOnlyDictionary`.